### DISCUSSION TITLE
Is there no way to run ElectronIM on Windows 7?

### DISCUSSION THREAD

**Discussion by 613TheEvil** [manusa/electronim#602] [Q&A]
I was using Rambox until today, it stopped supporting Win7. I switched to Franz, because Ferdi has... disappeared I guess. But it has issues. Looking for alternatives, I found ElectronIM. It looks promising but, The installer could not run, the one from the zip says it is not a valid Win32 application... Should I keep hoping or look elsewhere? Thanks to anyone kind enough to answer.

**Comment by manusa** [comment]
You can try to install NodeJS (a Windows 7 compatible version) and run with node  `npx -y electronim@latest` 

Other than that, it might be hard to get things working with an outdated OS.
